allennlp/modules/text_field_embedders/basic_text_field_embedder.py

```diff
@@ -69,75 +50,45 @@ def get_output_dim(self) -> int:
-                    f"between your token indexers and token embedders."
-                )
-                raise ConfigurationError(message)
-
-            elif self._token_embedders.keys() != text_field_input.keys():
-                # some other mismatch
-                message = "Mismatched token keys: %s and %s" % (
-                    str(self._token_embedders.keys()),
-                    str(text_field_input.keys()),
-                )
-                raise ConfigurationError(message)
+        if self._token_embedders.keys() != text_field_input.keys():
+            message = "Mismatched token keys: %s and %s" % (
+                str(self._token_embedders.keys()),
+                str(text_field_input.keys()),
+            )
+            raise ConfigurationError(message)
 
         embedded_representations = []
-        keys = sorted(embedder_keys)
-        for key in keys:
+        for key in self._ordered_embedder_keys:
             # Note: need to use getattr here so that the pytorch voodoo
             # with submodules works with multiple GPUs.
             embedder = getattr(self, "token_embedder_{}".format(key))
             forward_params = inspect.signature(embedder.forward).parameters
             forward_params_values = {}
+            missing_tensor_args = set()
             for param in forward_params.keys():
                 if param in kwargs:
                     forward_params_values[param] = kwargs[param]
+                else:
+                    missing_tensor_args.add(param)
 
             for _ in range(num_wrapping_dims):
                 embedder = TimeDistributed(embedder)
-            # If we pre-specified a mapping explictly, use that.
-            # make mypy happy
-            tensors: Union[List[Any], Dict[str, Any]] = None
-            if self._embedder_to_indexer_map is not None and key in self._embedder_to_indexer_map:
-                indexer_map = self._embedder_to_indexer_map[key]
-                if isinstance(indexer_map, list):
-                    # If `indexer_key` is None, we map it to `None`.
-                    tensors = [
-                        (text_field_input[indexer_key] if indexer_key is not None else None)
-                        for indexer_key in indexer_map
-                    ]
-                    token_vectors = embedder(*tensors, **forward_params_values)
-                elif isinstance(indexer_map, dict):
-                    tensors = {
-                        name: text_field_input[argument] for name, argument in indexer_map.items()
-                    }
-                    token_vectors = embedder(**tensors, **forward_params_values)
-                else:
-                    raise NotImplementedError
+
+            tensors: Dict[str, torch.Tensor] = text_field_input[key]
+            if len(tensors) == 1 and len(missing_tensor_args) == 1:
+                # If there's only one tensor argumnt to the embedder, and we just have one tensor to
```

Comment:
s/argumnt/argument/

Reply:
My `e` key on my laptop is starting to die...  Thanks for catching this.